### ISSUE TITLE
Error 404 on webpage.

### ISSUE THREAD

**Issue by BearHuntr** [hurricanehrndz/docker-subsonic#4]
I'm new to docker and this is my first attempt to use it on my Synology 1515+, so forgive me if I'm not providing the right info.  I set up Subsonic with the default ports and filled in the volumes and variables and it seems like it is running and scanning my music according to the logs, but I can't get the webpage to load.  I get an Error 404 page with a message of "org.springframework.jdbc.BadSqlGrammarException: PreparedStatementCallback; bad SQL grammar [select username, password, email, ldap_authenticated, bytes_streamed, bytes_downloaded, bytes_uploaded from user where username=?]; nested exception is java.sql.SQLException: Table not found in statement [select username, password, email, ldap_authenticated, bytes_streamed, bytes_downloaded, bytes_uploaded from user where username=?]"  I've run Subsonic successfully before on PCs and on the Synology via a package, but that stopped working with the recent update to Synology DSM 6.  I've completely uninstalled the previous Subsonic package and set the Subsonic home directory to a new empty folder and it created the db and other files there.  Any ideas how to fix this error?  Thanks!

**Comment by hurricanehrndz** [comment]
Which version are you using?

**Comment by hurricanehrndz** [comment]
Trying using the beta version it will be under docker-containers/subsonic or just subsonic. Let me know what happens.

**Comment by BearHuntr** [comment]
Sorry for not replying sooner, but I ended up figuring out what the problem was today.  Apparently, the  uninstall of the previous Subsonic package had not completed fully and that left port 4040 still in use.  When I rebooted the Synology, it freed up the port and I was able to redeploy the container and successfully get to the webpage.  Thanks for trying to help out, but the problem was not with your build.
